partials/archive-exhibitor/exhibitors.php

```diff
@@ -39,74 +46,50 @@
            </h2>
         </div>
 <?php 
-    while( $query->have_posts() ) {
-      $query->the_post();
-
-      $the_title = get_the_title();
-      $first_letter = $the_title[0];
-      // the first letter of the title
-
-      
-      if (!ctype_alpha($first_letter)) {
-        // if the first letter is not alphabetic, 
-        // set it to '#' for numbers and symbols
-        $first_letter = '#';
-      } else {
-        // otherwise make it uppercase
-        $first_letter = strtoupper($first_letter);
-      }
-      
+        while( $query->have_posts() ) {
+          $query->the_post();
 
-      if ($first_letter !== $section_letter) {
-        // if the first letter is not equal the current section letter
-        //then we reset the section letter, and start a new section
-
-        $section_letter = $first_letter;
+          if( has_term($exhibitor_slug, 'booth_level') ) {
+            include(locate_template('partials/archive-exhibitor/exhibitor.php'));
+          }
+        }
 ?>
       </div>
-      <div class="row">
-        <div class="col col-s col-s-12 col-m col-m-12 col-l col-l-12">
-          <h3><?php echo $section_letter; ?></h3>
-        </div>
-      </div>
-      <div class="row">
 <?php 
       }
+    }
 
-      $city = get_post_meta($post->ID, '_igv_exhibitor_city', true);
-      $link = get_post_meta($post->ID, '_igv_exhibitor_url', true);
-
-      if ($fair_year_id != $current_year_id) {
-?>
+    $section_letter = null;
 
-        <article <?php post_class('col col-s col-s-12 col-m col-m-12 col-l col-l-6 margin-bottom-micro'); ?> id="post-<?php the_ID(); ?>">
-        <?php if (!empty($link)) { ?>
-          <a href="<?php echo esc_url($link); ?>" target="_blank" rel="noopener noreferrer">
-            <h4 class="font-sans u-inline-block font-size-h3 border-underline"><?php the_title(); ?></h4>
-          </a>
-        <?php } else { ?>
-          <h4 class="font-sans u-inline-block font-size-h3"><?php the_title(); ?></h4>
-        <?php } ?>
-          <?php echo !empty($city) ? '<span class="font-size-h4">, ' . $city . '</span>': ''; ?> 
-        </article>
+    $project_slug = $fair_year . '-project';
+    $project_term =  get_term_by('slug', $project_slug, 'booth_level');
 
-<?php
-    } else {
+    if ($project_term) {
+      if ($project_term->count > 0) {
 ?>
+      <div class="row">
```

Comment:
as this section is the same as above except checking for and expressing as title a different taxonomy it would be better to use a function or similar as not to repeat code